AirCasting/Utils/Strings.swift

```diff
@@ -305,300 +439,501 @@ struct Strings {
             transmission of personal data – you may request to receive your personal data in a structured, commonly used, machine-readable format and to have that data sent to another controller. The above applies when data are processed upon your consent (Article 6(1)(a) GDPR) and/or upon contract (Article 6(1)(b) GDPR) and the data are processed by automated means;
             lodge a complaint to the President of the Office for Personal Data Protection – in the event of a breach by HabitatMap of the processing of your personal data, you may lodge a complaint with the relevant authority.
             You may contact HabitatMap with any request to exercise your rights by email or postal address shown at the beginning of this document or by other means of your choice. Details on how to file a complaint with the President of the Office of Personal Data Protection can be found at: https://uodo.gov.pl/pl/p/kontakt.
-
+ 
             Miscellaneous
-
+ 
             This Privacy Policy may be subject to updates and changes. In such case, HabitatMap will take steps to inform you of any such updates or changes by, for example, sending you a notification. Notwithstanding the foregoing, HabitatMap recommends that you review this page as often as possible.
-
- """).stringKey
+ 
+ """,
```

Comment:
general note: it's bette to put privacy policy on a website and redirect there or make a special endpoint to provide policy text. Reasons:
1) Currently to update the policy we need to push new version to the appstore
2) If we update this and push new version, users that doesn't update the app won't be subjected to the new rules. This can cause some legal issues

Maybe smth to talk thru with M.

Reply:
I will write a message to him and discuss it, thanks!